install/kubernetes/helm/istio/charts/egressgateway/templates/deployment.yaml

```diff
@@ -48,8 +48,10 @@ spec:
           - {{ .Values.service.name }}
           - --zipkinAddress
           - zipkin:9411
+        {{- if .Values.global.proxy.envoyStatsd }}
```

Comment:
more  like an  open question , should this parameter name be changed as well?  `--statsdUdpAddress`